moviedb_project/moviemanager/views.py

```diff
@@ -2,23 +2,35 @@
 from django.http import HttpResponse, HttpResponseRedirect,JsonResponse
 from .forms import NameForm
 from .models import NameFormModel
-import json, os,requests
+import json, os, requests
+from datetime import datetime
 # Create your views here.
 database = []
 def moviemanagerView(request):
     if request.method == 'POST':
         form = NameForm(request.POST)
         #year_model = Year(request.POST)
         if form.is_valid():
-            database.append(form.cleaned_data)
-            print(database)
-            print(type(database))
+            #Not initializing db here, instead storing cleaned data for manipulation
+            cleanForm = form.cleaned_data
+            print(cleanForm)
+            print(type(cleanForm))
             #obj = NameFormModel.objects.create(**form.cleaned_data)
-            form = NameForm()
-            movies_json = JsonResponse(database, safe = False)
+            movies_json = JsonResponse(cleanForm, safe = False)
+            #Always have to futz with dates for one reason or another. This prevents the db from having datetime() instead of your actual date.
+            cleanForm["date_watched_field"] = cleanForm["date_watched_field"].strftime("%m/%d/%Y")
```

Comment:
Needed to convert this before storing it as a json since there's no json datetime type. You can get away without using this logic when you go to a standardized DB (you're already using sqlite with your models)